kythe-verification/testdata/basic/ImportsRef.hs

```diff
@@ -0,0 +1,7 @@
+-- Verify the references to the imported modules
+module Imports where
+
+-- - @"Data.Set" ref/imports vname("containers-0.5.7.1:Data.Set", "", "", "", "haskell")
```

Comment:
I'm aware of it. Just couldn't find anything non-versioned except base. We can always switch to a base import if the test turns out to be noisy.